src/test/java/com/notably/logic/suggestion/SuggestionTestUtil.java

```diff
@@ -0,0 +1,109 @@
+
+        return model;
+    }
+
+    public static AbsolutePath getToCs2103t() {
+        return TypicalBlockModel.PATH_TO_CS2103T;
+    }
+
+    public static List<SuggestionItem> getExpectedSuggestionsToCs2103t() {
+        SuggestionItem cs2103t = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T.getStringRepresentation(), null);
+        SuggestionItem cs2103tLect = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_LECTURES.getStringRepresentation(), null);
+        SuggestionItem cs2103tTut = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIALS.getStringRepresentation(), null);
+        SuggestionItem cs2103tTut1 = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_1.getStringRepresentation(), null);
+        SuggestionItem cs2103tTut2 = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_2.getStringRepresentation(), null);
+
+        List<SuggestionItem> expectedSuggestions = new ArrayList<>();
+        expectedSuggestions.add(cs2103t);
+        expectedSuggestions.add(cs2103tLect);
+        expectedSuggestions.add(cs2103tTut);
+        expectedSuggestions.add(cs2103tTut1);
+        expectedSuggestions.add(cs2103tTut2);
+
+        return expectedSuggestions;
+    }
+
+    public static List<String> getExpectedInputsToCs2103t(String commandWord, boolean hasPrefixTitle) {
+        List<String> expectedInputs = new ArrayList<>();
+        String prefix = "";
+
+        if (hasPrefixTitle) {
+            prefix = "-t ";
+        }
+
+        expectedInputs.add(commandWord + " " + prefix + TypicalBlockModel.PATH_TO_CS2103T.getStringRepresentation());
+        expectedInputs.add(commandWord + " " + prefix
+                + TypicalBlockModel.PATH_TO_CS2103T_LECTURES.getStringRepresentation());
+        expectedInputs.add(commandWord + " " + prefix
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIALS.getStringRepresentation());
+        expectedInputs.add(commandWord + " " + prefix
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_1.getStringRepresentation());
+        expectedInputs.add(commandWord + " " + prefix
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_2.getStringRepresentation());
+
+        return expectedInputs;
+    }
+
+    /**
+     * Checks the correctness of the suggestions.
+     * @param expectedSuggestions The expected suggestions list.
+     * @param suggestions The actual suggestions list.
+     */
+    public static void testSuggestions(List<SuggestionItem> expectedSuggestions, List<SuggestionItem> suggestions) {
+        for (int i = 0; i < expectedSuggestions.size(); i++) {
+            SuggestionItem suggestion = suggestions.get(i);
+            SuggestionItem expectedSuggestion = expectedSuggestions.get(i);
+            assertEquals(expectedSuggestion.getProperty("displayText"), suggestion.getProperty("displayText"));
```

Comment:
Actually do you know that this can be done a lot simpler as such?

```java
assertEquals(expectedSuggestions.stream().map(s -> s.getProperty("displayText")).collect(Collectors.toList()), suggestions.stream().map(s -> s.getProperty("displayText")).collect(Collectors.toList()));
```

Reply:
thanks for the suggestion!